src/app-layout/visual-refresh-toolbar/toolbar/trigger-button/index.tsx

```diff
@@ -118,9 +119,10 @@ function TriggerButton(
       const relatedTarget = eventWithRelatedTarget?.relatedTarget;
       const isFromAnotherTrigger = relatedTarget?.dataset?.shiftFocus === 'awsui-layout-drawer-trigger';
       if (
-        isForSplitPanel || //for key navigation to button from breadcrumb
-        isFromAnotherTrigger || //for key navigation from another trigger button
-        !isForPreviousActiveDrawer //for when the drawer was not opened recently
+        (isForSplitPanel && !!relatedTarget) || // relatedTarget is null when split panel is closed
+        (!isForSplitPanel &&
+          (isFromAnotherTrigger || // for key navigation from another trigger button
+            !isForPreviousActiveDrawer)) // for when the drawer was not opened recently
```

Comment:
I'm not very knowledgeable about app layout so this part is not clear to me at all, what was the problem here and what is the expected behavior?

Reply:
this is for the focus event on the trigger buttons. If the trigger button is for split panel and if there is no related target in the event than the tooltip will be suppressed. 

This is for when the split panel (both bottom and side) are closed and and the focus then shifts back to the trigger button, in this case the tooltup wont show where focus from a click or key event it will show

Reply:
Let's arrange a pair programming session and try to refine it